### ISSUE TITLE
Error when trying to generate the documentation

### ISSUE THREAD

**Issue by JJ** [jmv74211/TFM_security_system_PI#36]
`pdflatex proyecto.pdf` stops with

```
! Package minted Error: You must invoke LaTeX with the -shell-escape flag.
```

Related to #35

**Comment by jmv74211** [comment]
Hello @JJ .

Minted is a package that allows formatting and highlighting source code in LaTeX.

I'm not currently using it in the documentation (although it's added in the section headers). The reason why it is added is that I intended to use it in later sections.

I'm going to comment this package for now, since I'm not using it, but in case I activate it again, the solution to this problem is described in the following link:  https://tex.stackexchange.com/questions/99475/how-to-invoke-latex-with-the-shell-escape-flag-in-texstudio-former-texmakerx

In my case, I'm using TeXstudio, and I had to do the following configuration:

Click on the following menu:

 ```
 Options > Configure TeXStudio > Commands 
 ```
 and change:

 ```
 pdflatex -synctex=1 -interaction=nonstopmode %.tex
 ```
into

```

pdflatex -synctex=1 -interaction=nonstopmode --shell-escape %.tex
```
The following image shows this change:

![imagen](https://user-images.githubusercontent.com/23462183/64487205-c7e1a980-d237-11e9-98fa-828642386290.png)

Regards.
